src/controller/mailing_controller.py

```diff
@@ -0,0 +1,61 @@
+from smtplib import SMTPResponseException
+from flask import request, Blueprint
+from flask import current_app as app
+from http import HTTPStatus
+
+from src.service.mailing.mailservice import (CreateReservationConfirmationMessage, CreateInvoiceMessage,
+                                             CreateResetPasswordMessage, CreatePaymentConfirmationMessage,
+                                             CreateActivationMessage, MailService)
+
+mailing = Blueprint('mailing', __name__, url_prefix='/mailing')
+
+def SelectMessageCreator(message_type, data_id, recipients):

```

Comment:
Trochę nie leży mi ta funkcja. Lepszym rozwiązaniem dla mnie było by użycie designu Factory albo Factory method, która na podstawię enuma -> message type wypluje nam odpowiedniego maila. Będzie to na pewno dużo czystrze rozwiązanie niż ta funkcja